src/app/components/forms/fireside/AppFormFiresideAdd.vue

```diff
@@ -10,43 +12,39 @@ import AppFormControlErrors from '../../../../_common/form-vue/AppFormControlErr
 import AppFormGroup from '../../../../_common/form-vue/AppFormGroup.vue';
 import { validateMaxLength, validateMinLength } from '../../../../_common/form-vue/validators';
 import { showErrorGrowl } from '../../../../_common/growls/growls.service';
+import { Realm } from '../../../../_common/realm/realm-model';
 import AppTranslate from '../../../../_common/translate/AppTranslate.vue';
 import { $gettext } from '../../../../_common/translate/translate.service';
-import AppPostTargetCommunity from '../../post/target/AppPostTargetCommunity.vue';
-import AppPostTargetAddCommunity from '../../post/target/_add/AppPostTargetAddCommunity.vue';
+import AppPostTargets from '../../post/AppPostTargets.vue';
 
 type FormModel = {
 	title: string;
-	community_id: number | null;
 	auto_feature: boolean;
 	add_community_as_cohosts: boolean;
 };
 
 const props = defineProps({
 	community: { type: Object as PropType<Community>, default: null },
+	realms: { type: Array as PropType<Realm[]>, default: () => [] },
```

Comment:
This list of realms is getting modified directly. If you add a realm through the selector and close the modals, any previously selected realms are still appearing when the FiresideAdd modal is opened again. Probably need to clone the list when initializing instead of using the `realms` ref everywhere.

Reply:
im not sure if this is the best way to fix this:
```
const realmsActual = ref<Realm[]>([...realms.value]);
```
after destructuring props. is this how we do things?